### PR TITLE
Update to reflect changes in SOR configuration

### PR DESCRIPTION
**Pull request by Bhavesh-Patel**

### CODE REVIEW

**Comment by Bhavesh-Patel** [comment]
[Implement data driven UI for triage data steps (Backend)](https://trello.com/c/JDzuL9BD/232-implement-data-driven-ui-for-triage-data-steps-backend)

File: docs/repairs-api/sor-engine.md [hunk 1]
```diff
@@ -38,205 +38,108 @@ Not all journey's have 3 tier's, e.g. Kitchen - Worktop
...
 ### Configuration
-The SoR engine is configured by specifying the SoR codes in [SoRConfig.json](https://github.com/City-of-Lincoln-Council/housing-repairs-online-api/blob/main/HousingRepairsOnlineApi/SoRConfig.json).
+The SoR engine is configured by specifying the `SOR_CONFIGURATION` environment variable.
 
-Each existing SoR code must exist and have a value.
+The configuration allows specifying the key value, display text and either options (used to gather additional information) or a SoR code.
 
-If an existing SoR code is removed, the mapping resolution will fail.
-If a new SoR code is added, it will have no effect. (The frontend would need to be updated to offer the new selection options.)
+The `SOR_CONFIGURATION` value is used to determine which options to offer at each tier (where, what and best description) and a SoR code for the combination of options.
```
Comment (agnieszkazbikowska-madetech) [inline docs/repairs-api/sor-engine.md:45]: `the best description`? (I'm not sure, I'm not a native speaker, @CatherineDeas would know best probably)

File: docs/repairs-api/sor-engine.md [hunk 2]
```diff
@@ -38,205 +38,108 @@ Not all journey's have 3 tier's, e.g. Kitchen - Worktop
...
+The display text is shown to the user within the frontend.  
+The value specified is used as a key to identify that option. All values need to be unique and do not need to be human readable as they won't be displayed.
+
+#### Early Exit Points
+An early exit point is an option selection which stops the user from continuing to make a repair request.
+The following describes the values to use for specific exit points:
+
+| Value                   | Description                                                     |
+|-------------------------|-----------------------------------------------------------------|
+| Emergency               | A option selected is identified as an emergency                 |
```
Comment (agnieszkazbikowska-madetech) [inline docs/repairs-api/sor-engine.md:55]: `An option`?

File: docs/repairs-api/sor-engine.md [hunk 3]
```diff
@@ -38,205 +38,108 @@ Not all journey's have 3 tier's, e.g. Kitchen - Worktop
...
+| UnableToBook            | A option which is not bookable via the service                  |
```
Comment (agnieszkazbikowska-madetech) [inline docs/repairs-api/sor-engine.md:56]: `An option`?

File: docs/repairs-api/sor-engine.md [hunk 4]
```diff
@@ -38,205 +38,108 @@ Not all journey's have 3 tier's, e.g. Kitchen - Worktop
...
+| NotEligibleNonEmergency | A option for which the Local Authority _may_ not be responsible |
```
Comment (agnieszkazbikowska-madetech) [inline docs/repairs-api/sor-engine.md:57]: `An option`?

**Review by agnieszkazbikowska-madetech — APPROVED** [review]

File: docs/repairs-api/sor-engine.md (hunk 2 above)
Comment (CatherineDeas) [reply docs/repairs-api/sor-engine.md:55]: yes

File: docs/repairs-api/sor-engine.md (hunk 3 above)
Comment (CatherineDeas) [reply docs/repairs-api/sor-engine.md:56]: yep

File: docs/repairs-api/sor-engine.md (hunk 4 above)
Comment (CatherineDeas) [reply docs/repairs-api/sor-engine.md:57]: indeed

File: docs/repairs-api/sor-engine.md (hunk 1 above)
Comment (CatherineDeas) [reply docs/repairs-api/sor-engine.md:45]: here no need to put "the" as it's "telegram" style

**Review by CatherineDeas — CHANGES_REQUESTED** [review]
Please see changes proposed by Agnieszka for the use of "an option"

**Review by agnieszkazbikowska-madetech — APPROVED** [review]

**Review by CatherineDeas — APPROVED** [review]

**Pull request merged**